components/rdv/EditionRdvForm.tsx

```diff
@@ -233,6 +235,15 @@ export function EditionRdvForm({
     else onChanges(false)
   })
 
+  function emailInvitationText(conseillerIsCreator: boolean) {
+    if (conseillerIsCreator) {
+      return `Intégrer ce rendez-vous à mon agenda via l’adresse mail suivante :
```

Comment:
```suggestion
      return `Intégrer ce rendez-vous à mon agenda via l’adresse e-mail suivante :
```
cf. https://zeroheight.com/2f9442f46/p/36c04e-lexique/b/75adeb